Makefile

```
@@ -36,9 +36,11 @@ check-vet:
 docker-test:
 	scripts/docker-compose-testing run -T --rm go-agent-tests make test
 
-.PHONY: test
+.PHONY: test-fips test
+test-fips: ARGS=-tags=requirefips
+test-fips: test
 test:
-	@for dir in $(shell scripts/moduledirs.sh); do (cd $$dir && go test -race -v -timeout=$(TEST_TIMEOUT) ./...) || exit $$?; done
+	@for dir in $(shell scripts/moduledirs.sh); do (cd $$dir && go test -race -v -timeout=$(TEST_TIMEOUT) $(ARGS) ./...) || exit $$?; done
```

- take care for `$dir` containing whitespace
- you don't need a subshell
```suggestion
	@for dir in $(shell scripts/moduledirs.sh); do cd "$$dir" && go test -race -v -timeout=$(TEST_TIMEOUT) $(ARGS) ./... || exit $$?; done
```

---

It's very unlikely we'll ever have a module with a whitespace in its name.

---

> /bin/sh: 1: cd: can't cd to ./module/apmazure

---

OK, I didn't thought of that exiting the subshell restores to the previous working directory :+1:

---

> It's very unlikely we'll ever have a module with a whitespace in its name.

Sure, but still let's do it the correct way. Code may be copied / spread.